src/main/java/org/spongepowered/common/command/registrar/BrigadierCommandRegistrar.java

```diff
@@ -84,36 +87,16 @@
         final PluginContainer container = PhaseTracker.getCauseStackManager().getCurrentCause().first(PluginContainer.class)
                 .orElseThrow(() -> new IllegalStateException("Cannot register command without knowing its origin."));
 
-        if (!this.hasVanillaRegistered && Launch.getInstance().getMinecraftPlugin() == container) {
-            final LiteralCommandNode<CommandSource> vanillaCommand = this.applyNamespace(container, command, false);
-            this.vanilla.add(vanillaCommand);
-            return vanillaCommand;
-        }
-
         return this.registerInternal(this,
                 container,
                 this.applyNamespace(container, command, false),
                 new String[0],
                 true).getSecond();
     }
 
-    public void completeVanillaRegistration() {
-        // At this point, let vanilla through anyway, if they haven't come through yet
-        // then they can register anyway.
-        this.hasVanillaRegistered = true;
-        try (final CauseStackManager.StackFrame frame = PhaseTracker.getCauseStackManager().pushCauseFrame()) {
-            final PluginContainer container = Launch.getInstance().getMinecraftPlugin();
-            frame.pushCause(container);
-            for (final LiteralCommandNode<CommandSource> node : this.vanilla) {
-                this.registerInternal(this, container, node, new String[0], true);
-            }
-        }
-        this.vanilla.clear();
-    }
-
     @Override
-    public TypeToken<LiteralArgumentBuilder<CommandSource>> handledType() {
-        return BrigadierCommandRegistrar.COMMAND_TYPE;
+    public @NonNull CommandRegistrarType<LiteralArgumentBuilder<CommandSource>> type() {
+        return TYPE;
```

Comment:
To do a @Zidane - this should be `BrigadierCommandRegistrar.TYPE`, fully qualifying all static access, even if in the same class.